### DISCUSSION TITLE
Task lists: converting a checkbox line with image or link loses the image or link

### DISCUSSION THREAD

**Discussion by tungv** [community/community#4516] [Projects and Issues]
If I use markdown to embed images or links in an item of a check list, then converting it to an issue will lose the embedded images or links.

Expected/Suggestion:
- capture the embedded links or images in the issue content, or
- keep the embedded items in the check list next to the newly created issue id

**Comment by rileytaylor** [comment]
I experienced this as well. Like the OP mentioned, it's probably just since it's all being copy pasted into the title which doesn't support markdown, but as long as it's pasted as `[info](URL)` in the title i'd be happy to edit the title and paste back into the content. I just lost a big long paragraph with a lot of helpful links and had to spend a good hour finding those again since there is no way to undo this operation.

**Comment by arthurljones** [comment]
I've also recently been bitten by this when links to designs and specs were lost when converting a task list item to an issue. This could be worked around if their was either a way to undo the action (which only takes a single click) or the information was retained in some fashion.
